mmdet/models/necks/ct_resnet_neck.py

```diff
@@ -0,0 +1,87 @@
+import math
+
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule, auto_fp16
+
+from mmdet.models.builder import NECKS
+
+
+@NECKS.register_module()
+class CTResNetNeck(BaseModule):
+    """The neck used in `CenterNet <https://arxiv.org/abs/1904.07850>`_ for
+    object classification and box regression.
+
+    Args:
+         in_channel (int): Number of input channels.
+         num_filters (List[int]): Number of filters per stage.
+         num_kernels (List[int]): Number of kernels per stage.
```

Comment:
`kernel_sizes` and `kernel sizes of ConvTranspose2d  per stage` would be more clear.

Reply:
I plan to refer to mmpose writing, such as `num_deconv_filters` and `num_deconv_kernels`.

Reply:
But the meaning of 'number of filters' or 'kernels' is totally different from 'kernel size'. 'Number of filters' is more like 'output channels' because the number of filters in convolution is determined by the output channels.